packages/wdio-devtools-service/src/index.ts

```diff
@@ -55,6 +57,15 @@ export default class DevToolsService implements WebdriverIO.HookFunctions {
     private _traceGatherer?: TraceGatherer
     private _devtoolsGatherer?: DevtoolsGatherer
 
+    constructor (
+        serviceOptions: ServiceOptions,
+        caps: WebDriver.Capabilities,
+        config: WebdriverIO.Config,
+        private _browser: WebdriverIO.BrowserObject | WebdriverIO.MultiRemoteBrowserObject
+    ) {
+        this._browser = _browser
```

Comment:
this can be removed as `_browser` is already assigned and you are overwriting it with the same thing